hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -36,6 +36,8 @@ public record ContractsConfig(
         @ConfigProperty(value = "localCall.estRetBytes", defaultValue = "32") @NetworkProperty int localCallEstRetBytes,
         @ConfigProperty(defaultValue = "true") @NetworkProperty boolean allowCreate2,
         @ConfigProperty(defaultValue = "false") @NetworkProperty boolean allowAutoAssociations,
+        @ConfigProperty(value = "lazyCreation.enabled", defaultValue = "true") @NetworkProperty
```

Comment:
All properties in this `ContractsConfig` will be prefixed by `contracts`'; so this property would be,
```
contracts.lazyCreation.enabled=true
```
In any case, this property already exists in `LazyCreationConfig`; so we can remove this change.

I think `TraceabilitySuite` is not passing simply due to missing records.